### ISSUE TITLE
fs-file-max and vm.swappiness sensible defaults?

### ISSUE THREAD

**Issue by bradthurber** [saltstack-formulas/sysctl-formula#16]
The current defaults.yml (below) sets default params fs-file-max:100000 and vm.swappiness:20 . Are these truly good sensible defaults that we should always be applying? I am the opposite of an expert on these params, but looking at upvoted stackoverflow answers, I think these defaults should be removed from the formula if possible.

[swappiness Q&A on stackexchange](http://unix.stackexchange.com/questions/88693/why-is-swappiness-set-to-60-by-default)
"The default value [60] has most likely been chosen as an approximate middleground between these two extremes. As with any performance parameter, adjusting vm.swappiness should be based on benchmark data comparable to real workloads, not just a gut feeling"

[fs-file-max Q&A on stackoverflow](http://stackoverflow.com/questions/6180569/need-to-calculate-optimum-ulimit-and-fs-file-max-values-according-to-my-own-se)
"For fs.file-max, I think in almost all cases you can just leave it alone" and
"The 2.6 kernel automatically tunes fs.file-max based on available memory"

``` YAML
# -*- coding: utf-8 -*-
# vim: ft=yaml
sysctl:
  pkg: procps-ng
  config:
    location: '/etc/sysctl.d'
  lookup:
    params:
      -
        name: fs.file-max
        value: 100000
        config: fs.conf
      -
        name: vm.swappiness
        value: 20
```

**Comment by bradthurber** [comment]
Fixed by #14

I think @babilen and I woke up today and decided to work on the same thing :smiley:

**Comment by wwentland** [comment]
Haha, that's amazing :)
